prov/verbs/src/verbs_info.c

```diff
@@ -561,6 +561,7 @@ static int fi_ibv_get_device_attrs(struct ibv_context *ctx, struct fi_info *info
 	info->domain_attr->max_ep_rx_ctx 	= device_attr.max_qp;
 	if (info->ep_attr->type == FI_EP_RDM)
 		info->domain_attr->cntr_cnt	= device_attr.max_qp * 4;
+	info->domain_attr->mr_iov_limit		= 1;
```

Comment:
We have verbs_domain_attr const variable where this can be set. The variable is later copied when creating an fi_info. This function is limited to setting the device specific attributes.